skimage/color/colorconv.py

```diff
@@ -1565,32 +1480,34 @@ def separate_stains(rgb, conv_matrix, *, channel_axis=-1):
     Examples
     --------
     >>> from skimage import data
-    >>> from skimage.color import separate_stains, hdx_from_rgb
+    >>> from skimage.color import separate_stains, stain_color_matrix
     >>> ihc = data.immunohistochemistry()
-    >>> ihc_hdx = separate_stains(ihc, hdx_from_rgb)
+    >>> m = stain_color_matrix(('Hematoxylin', 'DAB'))
+    >>> ihc_hd = separate_stains(ihc, m)
     """
     rgb = _prepare_colorarray(rgb, force_copy=True, channel_axis=-1)
     np.maximum(rgb, 1E-6, out=rgb)  # avoiding log artifacts
     log_adjust = np.log(1E-6)  # used to compensate the sum above
 
-    stains = (np.log(rgb) / log_adjust) @ conv_matrix
+    stains = (np.log(rgb) / log_adjust) @ linalg.pinv(color_matrix)
 
     np.maximum(stains, 0, out=stains)
 
     return stains
 
 
 @channel_as_last_axis()
-def combine_stains(stains, conv_matrix, *, channel_axis=-1):
-    """Stain to RGB color space conversion.
+def combine_stains(stains, color_matrix, *, channel_axis=-1):
+    """Mix stains into a simulated RGB brightfield image.
 
     Parameters
     ----------
-    stains : (..., 3, ...) array_like
-        The image in stain color space. By default, the final dimension denotes
-        channels.
-    conv_matrix: ndarray
-        The stain separation matrix as described by G. Landini [1]_.
+    stains : (..., 1/2/3, ...) array_like
+        The image in stain color space, with one to three channels. The
+        number of channels must match the size of `color_matrix`.
+        By default, the final dimension denotes channels.
+    color_matrix: ndarray
+        The stain color matrix as constructed with `stain_color_matrix()`.
```

Comment:
```suggestion
        The stain color matrix as constructed with
        `skimage.color.stain_color_matrix`.
```
This syntax will create a hyperlink in the online docs as in, for example: https://github.com/scikit-image/scikit-image/blob/660899ae7027c68a6aec085794d2a5bcc440fd32/skimage/measure/_marching_cubes_lewiner.py#L279